src/khoj/interface/web/config.html

```diff
@@ -8,62 +8,127 @@ <h2 class="section-title">Plugins</h2>
             <div class="card">
                 <div class="card-title-row">
                     <img class="card-icon" src="/static/assets/icons/github.svg" alt="Github">
-                    <h3 class="card-title">Github</h3>
+                    <h3 class="card-title">
+                        Github
+                        {% if current_config.content_type.github %}
```

Comment:
Hey, @trippleflux, thanks for testing out this change. Good catch, looks like I need to add some null checks here.

Reply:
Should be addressed with [bf8914d](https://github.com/khoj-ai/khoj/pull/269/commits/bf8914d0c8911e73a5c776b71afdf0df1760197e)